src/lib/plot/lobf.typ

```diff
@@ -0,0 +1,64 @@
+#import "line.typ": add,
+#import "mark.typ"
+
+#let lobf-modes = (
+  "linear",
+)
+
+#let add-lobf(
```

Comment:
I would prefer `add-fit` and `fit.typ` as filename.

Reply:
Committed